docs/advanced-features/module-path-aliases.md

```diff
@@ -93,3 +93,35 @@ export default function HomePage() {
   )
 }
 ```
+
+The additional `paths` are reletive to the `baseUrl`. For example:
+
+```json
+// tsconfig.json or jsconfig.json
+{
+  "compilerOptions": {
+    "baseUrl": "src/components/",
+    "paths": {
+      "@styles/*": ["../styles/*"],
+      "~/*": ["../*"],
+      "/*": ["../../*"]
```

Comment:
I think this example may cause confusion as `baseUrl` shouldn't really be used in this way and more so should point to the packages base. Potentially it could show a nested package as the `baseUrl` and including shared utils in paths?

Reply:
Oh ohk. I am pretty new to NextJS, so I was not aware this is a bad practice. It does seem to be working in my project, but I think we could just remove this example. 

I do think that it would still be helpful to mention that the paths will start at the baseUrl location instead of the root.

Reply:
Yeah sounds good to keep just the note and maybe strip the example 👍

Reply:
Ok, I have removed the example and also added a commit with a better example in case you want to include one.